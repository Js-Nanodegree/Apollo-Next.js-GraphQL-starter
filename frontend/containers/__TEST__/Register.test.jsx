import renderer from 'react-test-renderer';
import RegisterContainer from '../Register';

describe('PreRegister container', () => {
  it('should match snapshot', () => {
    const handleChange = jest.fn();
    const register = jest.fn();

    const tree = renderer
      .create(
        <RegisterContainer
          handleChange={handleChange}
          register={register}
          firstName=''
          lastName=''
          password=''
          passwordRepeat=''
          called={false}
          loading={false}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
