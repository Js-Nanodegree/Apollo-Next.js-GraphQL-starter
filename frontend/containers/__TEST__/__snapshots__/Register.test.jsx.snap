// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreRegister container should match snapshot 1`] = `
<div
  className="qepopj-0 gBRnsH"
>
  <form
    className="cw1g19-0 cGYWuk"
    onSubmit={[Function]}
  >
    <h1>
      Complete your registration
    </h1>
    <div
      className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        First name
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-fullWidth-45 MuiInput-fullWidth-30 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoComplete="first-name"
          className="MuiInputBase-input-46 MuiInput-input-31"
          disabled={false}
          name="firstName"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
          value=""
        />
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        Last name
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-fullWidth-45 MuiInput-fullWidth-30 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoComplete="last-name"
          className="MuiInputBase-input-46 MuiInput-input-31"
          disabled={false}
          name="lastName"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
          value=""
        />
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        Password
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-fullWidth-45 MuiInput-fullWidth-30 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoComplete="new-password"
          className="MuiInputBase-input-46 MuiInput-input-31 MuiInputBase-inputType-49 MuiInput-inputType-34"
          disabled={false}
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="password"
          value=""
        />
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        Repeat password
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-fullWidth-45 MuiInput-fullWidth-30 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoComplete="repeat-password"
          className="MuiInputBase-input-46 MuiInput-input-31 MuiInputBase-inputType-49 MuiInput-inputType-34"
          disabled={false}
          name="passwordRepeat"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="password"
          value=""
        />
      </div>
    </div>
    <br />
    <br />
    <button
      className="MuiButtonBase-root-79 MuiButtonBase-disabled-80 MuiButton-root-53 MuiButton-text-55 MuiButton-flat-58 MuiButton-disabled-73 MuiButton-fullWidth-78"
      disabled={true}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="-1"
      type="submit"
    >
      <span
        className="MuiButton-label-54"
      >
        REGISTER
      </span>
    </button>
  </form>
</div>
`;
